doc/AIRTABLE.md

```diff
@@ -69,3 +69,22 @@ To do anything in AirTable, always take the following steps:
 6. Navigate to the [eligibilityPaths](https://airtable.com/tblHt9ZuCBwsplYw2/viwWR17CgrYqiNa0B) sheet.
 7. Determine if the new path you wrote down already exists. If it exists, add the benefit to the benefits column for that row. If it doesn't exist, create a new row and add the benefit.
 8. Delete the benefit from the row for the incorrect eligibility path
+
+### Adding a need, associating it with a benefit
+
+1. Navigate to the [needs sheet](https://airtable.com/tblY7TpDsGR5I3SaN/viwQOzMWARRdWFks3) in AirTable
+2. Add a new row at the bottom and give the need an english name and french name
+3. Select the cell under the benefits column and click +
+4. Start typing the names of the benefits you want the need to be associated with
+
+To change the benefits associated with an existing need, click the "expand" icon on the right of the cell and then you will have the option to either unlink existing records or link new ones.
+
+### How to add a question and answers
```

Comment:
I think there may be some extra logic missing in these steps because I can't seem to get my test question to appear in the site.